servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingStreamingHttpService.java

```diff
@@ -27,7 +28,8 @@
      *
      * @param ctx Context of the service.
      * @param request to handle.
-     * @param response to send to the client.
+     * @param response to send to the client. The implementation of this method is responsible for calling
```

Comment:
@idelpivnitskiy - this change went in previously and we discussed the trade-offs highlighted in the commit message. Note that I updated the gRPC javadocs to clarify ownership in https://github.com/apple/servicetalk/pull/1405.